.github/workflows/maven-verify.yml

```diff
@@ -21,8 +21,13 @@ on:
   push:
   pull_request:
 
+permissions:
+  contents: read
+
 jobs:
   build:
+    permissions:
+      contents: none
```

Comment:
and next set to none ... why not on global level?

Reply:
Agreed. Let me fix it.